tests/validation/convergence_test.hpp

```diff
@@ -2,10 +2,11 @@
 
 #include <util/filter.hpp>
 #include <util/rangeutil.hpp>
+#include <cell.hpp>
 
 #include <json/json.hpp>
 
-#include "gtest.h"
+#include "../gtest.h"
```

Comment:
I have fixed the `CMakeLists.txt` file to not include the base path, and adjusted all files in the tests path to use `../xyz.hpp`.